### ISSUE TITLE
Need support for additional file types on login pages

### ISSUE THREAD

**Issue by mdorazio** [coova/coova-chilli#353]
I'm trying to make a custom login page that looks nice in addition to being functional.  However, I cannot find a way to make Chilli support svg files, ttf, woff, etc.  Right now I get invalid file extension errors if I try to include these in the login page html.

Is there a way to add supported file types beyond the standard gif/jpg/css, etc.?  If not, please consider adding this as a feature in the future.

**Comment by irongomme** [comment]
This is not related to coova, but this is more html and browser compatibility .... 

Another way is to host your login page in a webserver

**Comment by pinkra** [comment]
No, it's related to coovachilli, because it handle only certain types of extensions for urls.

**Comment by mdorazio** [comment]
From the comment in defaults explaining HS_WWWDIR functionality:

#   Directory specifying where internal web pages can be served
#   by chilli with url /www/<file name>. Only extentions like .html
#   .jpg, .gif, .png, .js are allowed. See below for using .chi as a
#   CGI extension.

I would like to extend this to include additional accepted file types on the local coovachilli server.
